apiary.apib

```diff
@@ -5396,3 +5396,26 @@ From now on you won't be able to sign in to the **Direct Access** for specific p
             X-StorageApi-Token: your_token
 
 + Response 204
+
+###Reset credentials for backend [PUT /v2/storage/direct-access/{backend}/reset-password]
+Resets password for **Direct Access** backend. In response you will get url address for create new password.
+The password will be reset only for the project for which the **storage api token** is created.
```

Comment:
```suggestion
The password reset link is valid for 4 hours. The previous password is still valid until it's changed using provided URL. If the request times out, you can re-request new password change URL. 
```
Vůbec nechápu, co ta původní věta měla říct. Je to jako kdybysme upozorňovali, že to vytvoří credentials jen v tom projektu, kde vytváříš credentials. Nebo že když si uděláš storage token, tak že jde využít jen v projektu, kde sis ho vytvořil. Vůbec bych to tam nedával, jen to zmate IMHO.